web/src/components/dateTime.js

```diff
@@ -0,0 +1,34 @@
+import {Typography} from '@material-ui/core';
+import React, {Component} from 'react';
+import PropTypes from 'prop-types';
+
+class Datetime extends Component {
+  constructor(props) {
+    super(props);
+    this.state = {
+      schedule: this.props.children,
+    };
+  }
+  render() {
+    const date = new Date(this.state.schedule.startDateTime);
+    const month = date.toLocaleString('US', {month: 'long'}).substring(0, 3);
+    const year = date.toLocaleString('US', {year: 'numeric'});
+    const day = date.toLocaleString('US', {day: '2-digit'});
```

Comment:
Does this mean 1~9 will be 01, 02, ... ,09 ?

Reply:
Yes. It's right